descartes_planner/src/dense_planner.cpp

```diff
@@ -31,10 +31,10 @@ DensePlanner::~DensePlanner()
 
 }
 
-bool DensePlanner::initialize(descartes_core::RobotModelConstPtr &model)
+bool DensePlanner::initialize(descartes_core::RobotModelConstPtr model)
 {
   planning_graph_ = boost::shared_ptr<descartes_planner::PlanningGraph>(
-      new descartes_planner::PlanningGraph(model));
+      new descartes_planner::PlanningGraph(std::move(model)));
```

Comment:
Moving the shared pointer avoids an atomic reference access and write. It also shows that we're just passing that value along. It's a small pro, but I think it is 'most' correct. Omitting the move wouldn't hurt the logic of the code.